src/sqlfluff/cli/commands.py

```diff
@@ -209,6 +209,13 @@ def core_options(f: Callable) -> Callable:
             "the standard configuration files. N.B. cfg format is required."
         ),
     )(f)
+    f = click.option(
```

Comment:
Is this PR still in progress? I would expect to see code in `commands.py` that uses the `encoding` value, but I don't. (I should note that I don't fully understand how `@core_options` works.)

Reply:
@barrywhart the PR is done, it's quite a simple one.

The core options get applied as a decorator to lint, fix, and parse. It is passed to these functions via `**kwargs`:
![image](https://user-images.githubusercontent.com/80432516/143791963-db3f1a5f-399e-4db2-a9a3-a923c01511a7.png)

the `kwargs` then gets applied to the config in `get_config` via `overrides`:
![image](https://user-images.githubusercontent.com/80432516/143792035-b91cd7c9-53cc-4841-8aab-03d8f43a2de1.png)

and that's it, the rest has already been done in prior config work 👍